File: fitrbackend/src/main/java/com/example/fitrbackend/dto/CreateSetLogRequest.java
package com.example.fitrbackend.dto;

import lombok.Getter;

@Getter
public class CreateSetLogRequest {
    private int sets;
    private int averageReps;
    private float averageWeight;
    private Long averageDurationSeconds;
    private float averageDistance;
    private float averageCalories;
}

File: fitrfrontend/fitrfrontend/Models/DTO/CreateSetLogRequest.swift
//
//  CreateSetLogRequest.swift
//  fitrfrontend
//
//  Created by Ambrose Blay on 2/17/26.
//

import Foundation

struct CreateSetLogRequest: Codable {
    let sets: Int
    let averageReps: Int
    let averageWeight: Float
    let averageDurationSeconds: Int64?
    let averageDistance: Float
    let averageCalories: Float
}
